bids/analysis/transformations/compute.py

```diff
@@ -35,6 +36,25 @@ def _transform(self, var, model='spm', derivative=False, dispersion=False,
 
         if isinstance(var, SparseRunVariable):
```

Comment:
this logic should apply to all variables not just sparse ones, unless var is Y and not an X variable. where X are things that go into the design matrix and Y is the functional run. i'll post more thoughts in response to @tyarkoni comment in the discussion thread.

Reply:
nevermind - i think this is fine to apply to this type of variable. longer discussion in the other thread coming up shortly.